commons/src/main/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerConfig.java

```diff
@@ -138,6 +157,10 @@ ObjectStorageFactory objectStorageFactory() {
         return objectFactory;
     }
 
+    ChunkCache chunkCache() {
+        return getConfiguredInstance(CHUNK_CACHE_CONFIG, ChunkCache.class);
```

Comment:
Ok, now I realize it wasn't that great a suggestion to use get `getConfiguredInstance`, because it doesn't have a way to cut off the prefix from the names. Sorry for this suggestion :( Let's return to some
```java
        final ChunkCache chunkCache = Utils.newInstance(
            getClass(CHUNK_CACHE_CONFIG), ChunkCache.class
        );
        chunkCache.configure(this.originalsWithPrefix(CHUNK_CACHE_PREFIX));
        return chunkCache;
```

Reply:
Did a rollback to previous version since null check still makes sense